integration/spark/src/test/scala/org/apache/carbondata/integration/spark/testsuite/complexType/TestComplexDataType.scala

```diff
@@ -765,6 +766,36 @@ class TestComplexDataType extends QueryTest with BeforeAndAfterAll {
     assertResult("Unsupported operation on Complex data type")(arrayException.getMessage)
   }
 
+  test("testing the long string properties for complex columns in main table") {
+    sql("drop table if exists complex1")
+    sql("drop table if exists complex2")
+    sql("drop table if exists complex3")
+    sql("create table" +
+        " complex1 (a int, arr1 array<string>) " +
+        "stored as carbondata")
```

Comment:
please correct the style here, try to include maximum lines in one line, please correct for this and after test case end, leave a line gap

Reply:
done